src/SourceBuild/tarball/content/test/Microsoft.DotNet.SourceBuild.SmokeTests/BaselineHelper.cs

```diff
@@ -34,22 +34,22 @@ public static void Compare(string baselineFileName, IOrderedEnumerable<string> a
             Assert.Null(message);
         }
 
-        public static void Compare(string baselineFileName, string actual, ITestOutputHelper outputHelper)
+        public static void CompareContents(string baselineFileName, string actualContents, ITestOutputHelper outputHelper)
         {
             string baselineFilePath = GetBaselineFilePath(baselineFileName);
             string baseline = File.ReadAllText(baselineFilePath);
```

Comment:
There doesn't appear to be a need to read this file anymore.  This may be a leftover from some of my previous refactoring.